ppml/services/remote-attestation-service/docker/Dockerfile

```diff
@@ -0,0 +1,101 @@
+    wget -P ${SPARK_HOME}/jars/ https://repo1.maven.org/maven2/org/slf4j/slf4j-reload4j/1.7.35/slf4j-reload4j-1.7.35.jar && \
+    wget -P ${SPARK_HOME}/jars/ https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-api/2.17.1/log4j-api-2.17.1.jar && \
+    wget -P ${SPARK_HOME}/jars/ https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-core/2.17.1/log4j-core-2.17.1.jar && \
+    wget -P ${SPARK_HOME}/jars/ https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-slf4j-impl/2.17.1/log4j-slf4j-impl-2.17.1.jar
+    
+ADD https://raw.githubusercontent.com/intel-analytics/BigDL/main/docker/bigdl-k8s/log4j2.xml ${SPARK_HOME}/conf/log4j2.xml
+
+# stage.3 remote attestation service
+FROM ubuntu:20.04
+
+ARG SPARK_VERSION
+ARG BIGDL_VERSION
+ENV BIGDL_VERSION       ${BIGDL_VERSION}
+ENV SPARK_VERSION       ${SPARK_VERSION}
+ENV JAVA_HOME           /opt/jdk8
+ENV SPARK_HOME          /opt/spark-${SPARK_VERSION}
+ENV BIGDL_HOME          /opt/bigdl-${BIGDL_VERSION}
+ENV PATH                ${JAVA_HOME}/bin:${SPARK_HOME}/bin:${PATH}
+
+# Step 1. Install requirement tools
+RUN apt-get update --fix-missing && \
+    env DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y tzdata && \
+    apt-get install -y vim autoconf automake build-essential cmake curl debhelper git libcurl4-openssl-dev libprotobuf-dev libssl-dev libtool lsb-release ocaml ocamlbuild protobuf-compiler python wget libcurl4 libprotobuf17 libssl1.1 make kmod g++ libjsoncpp-dev uuid-dev openjdk-8-jdk && \
+# Step 2. Install Intel SGX SDK
+    mkdir /opt/intel && cd /opt/intel && \
+    wget https://download.01.org/intel-sgx/sgx-dcap/1.15/linux/distro/ubuntu20.04-server/sgx_linux_x64_sdk_2.18.100.3.bin && \
+    chmod a+x ./sgx_linux_x64_sdk_2.18.100.3.bin && \
+    printf "no\n/opt/intel\n"|./sgx_linux_x64_sdk_2.18.100.3.bin && \
+    . /opt/intel/sgxsdk/environment && \
+# Step 3. Install DCAP packages and set PCCS
+# DCAP repository setup
+    wget https://download.01.org/intel-sgx/sgx-dcap/1.15/linux/distro/ubuntu20.04-server/sgx_debian_local_repo.tgz && \
+    tar xzf sgx_debian_local_repo.tgz && \
+    echo 'deb [trusted=yes arch=amd64] file:///opt/intel/sgx_debian_local_repo focal main' | tee /etc/apt/sources.list.d/intel-sgx.list && \
+    wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add - && \
+    apt-get update --fix-missing && \
+    apt-get install -y libsgx-enclave-common-dev libsgx-ae-qe3 libsgx-ae-qve libsgx-urts libsgx-dcap-ql libsgx-dcap-default-qpl libsgx-dcap-quote-verify-dev libsgx-dcap-default-qpl-dev libsgx-quote-ex-dev libsgx-uae-service libsgx-ra-network libsgx-ra-uefi libsgx-dcap-ql-dev libtdx-attest libtdx-attest-dev
+
+COPY --from=builder /opt/jdk /opt/jdk8
+COPY --from=builder /opt/spark-${SPARK_VERSION} /opt/spark-${SPARK_VERSION}
+# COPY --from=bigdl ${BIGDL_HOME} ${BIGDL_HOME}
+
+RUN cd / && wget -c https://downloads.lightbend.com/scala/2.11.8/scala-2.11.8.tgz && \
```

Comment:
do we need scala in this image?

Reply:
yes, to start `BigDLRemoteAttestationService`